src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/ConfirmationStatementService.java

```diff
@@ -86,6 +86,21 @@ public ConfirmationStatementService(CompanyProfileService companyProfileService,
         }
     }
 
+    public ResponseEntity<Object> getConfirmationStatement(String submissionId) {
+        // Check Submission exists
+        var submission = confirmationStatementSubmissionsRepository.findById(submissionId);
+
+        if (submission.isPresent()) {
+            // Save updated submission to database
```

Comment:
i think this comment is wrong, doesn't look like we are saving here, only returning.